week3/assignment/src/main.jsx

```diff
@@ -0,0 +1,12 @@
+import { Global, ThemeProvider } from '@emotion/react';
+import { createRoot } from 'react-dom/client';
+import App from './App.jsx';
+import GlobalStyle from './styles/global.js';
+import theme from './styles/theme.js';
+
+createRoot(document.getElementById('root')).render(
+  <ThemeProvider theme={theme}>
+    <Global styles={GlobalStyle} />
+    <App />
+  </ThemeProvider>,
```

Comment:
p3: `StrictMode`를 사용하지 않은 이유가 따로 있을까요? 개발 중 잠재적인 문제를 감지하기 위해서 `StrictMode`를 사용하는 것이 어떨까요? `StrictMode`의 역할 관련한 리액트 공식문서 링크 남깁니다.
https://ko.react.dev/reference/react/StrictMode

```suggestion
  <StrictMode>
    <ThemeProvider theme={theme}>
      <Global styles={GlobalStyle} />
      <App />
    </ThemeProvider>
  </StrictMode>
```